### PR TITLE
Class Project

### PR DESCRIPTION
**Pull request by Emmacsea**
Project to create a sign up and login page for an organization

### CODE REVIEW

File: bank.html
Comment (Ademola101) [inline bank.html:1]: format your html code with Ctrl+Shift+P and then search for and execute “Format Document

File: style.css [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    padding: 1em 3.5em;
+    font-size: 1em;
+}
+
+.body .google-icon img {
+    width: 18px;
+    height: 100%;
+}
+
+/* .g-icon {
```
Comment (Ademola101) [inline style.css]: remove the comment

File: bank.html
Comment (Emmacsea) [reply bank.html:1]: Done

File: style.css (hunk 1 above)
Comment (Emmacsea) [reply style.css]: Done

**Review by Ademola101 — APPROVED** [review]
merge to the main branch

**Pull request merged**